### PR TITLE
OPENEUROPA-2965: Allowing graph results to be altered when loading RDF Skos entities

### PR DESCRIPTION
**Pull request by sergepavle**

### CODE REVIEW

File: modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  }
+
+  /**
+   * Process of Graph Results handler.
+   *
+   * @param \Drupal\rdf_skos\Event\SkosProcessGraphResultsEvent $event
+   *   Response event.
+   */
+  public function onProcessGraphResults(SkosProcessGraphResultsEvent $event) {
+    $result = $event->getResults();
```
Comment (upchuk) [inline modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php:51]: `$results`

File: modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        }
+      }
+    }
+    $event->setResults($result);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
```
Comment (upchuk) [inline modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php:81]: Can you move this please to the top of the class? It's pretty standard to quickly see what the class subscribes to.

File: modules/rdf_skos_language_mapping/tests/src/Kernel/RdfSkosLanguageMappingKernelTest.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\rdf_skos_language_mapping\Kernel;
+
+use Drupal\language\Entity\ConfigurableLanguage;
+use Drupal\Tests\rdf_skos\Kernel\RdfSkosKernelTestBase;
+
+/**
+ * Tests the RDF SKOS entities.
```
Comment (upchuk) [inline modules/rdf_skos_language_mapping/tests/src/Kernel/RdfSkosLanguageMappingKernelTest.php:11]:
```
Tests the mapping between Drupal language codes and the ones in Skos entities.
```

File: modules/rdf_skos_language_mapping/rdf_skos_language_mapping.info.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+name: RDF SKOS language mapping
```
Comment (upchuk) [inline modules/rdf_skos_language_mapping/rdf_skos_language_mapping.info.yml:1]: Let's add a README.md to this module.

File: runner.yml.dist [hunk 5]
```diff
@@ -23,7 +23,7 @@ drupal:
     - "./vendor/bin/drush config-set system.theme admin seven -y"
     - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
     # Enable the modules.
-    - "./vendor/bin/drush en rdf_skos -y"
+    - "./vendor/bin/drush en rdf_skos content_translation rdf_skos_language_mapping -y"
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply runner.yml.dist:26]: Was used for behat tests.

File: modules/rdf_skos_language_mapping/src/Form/LanguageMappingForm.php [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+      $t_args = [
+        '%language' => $language->getName(),
+        '%langcode' => $language->getId(),
+      ];
+
+      $form['language_mapping'][$langcode] = [
+        '#type' => 'textfield',
+        '#title' => $this->t('RDF SKOS %language (%langcode)', $t_args),
+        '#maxlength' => 64,
+        '#default_value' => isset($mapped_langcodes[$langcode]) ? $mapped_langcodes[$langcode] : substr($langcode, 0, 2),
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply modules/rdf_skos_language_mapping/src/Form/LanguageMappingForm.php:88]: It makes possible automatically have `pt` for Portugues language.

File: modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php (hunk 2 above)
Comment (sergepavle) [reply modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php:81]: I moved this method to place just after `__construct` (traditionally left __construct in the first place).

Comment (upchuk) [reply modules/rdf_skos_language_mapping/src/EventSubscriber/RdfSkosLanguageMappingSubscriber.php]: Exactly. Thank you!

**Pull request merged**